src/frontend/src/defaults/formatters.tsx

```diff
@@ -21,13 +43,21 @@ interface formatCurrencyOptionsType {
  * - digits: Maximum number of significant digits (default = 10)
  */
 export function formatCurrency(
-  value: number | null,
+  value: number | string | null | undefined,
   options: formatCurrencyOptionsType = {}
 ) {
-  if (value == null) {
+  if (value == null || value == undefined) {
     return null;
   }
 
+  value = parseFloat(value.toString());
```

Comment:
Question: Is there a way to only convert the value if it is not a number? Seems redundant to do it this way but not sure how much overhead this creates